datafusion/core/tests/dataframe/mod.rs

```diff
@@ -2447,8 +2448,8 @@ async fn test_count_wildcard_on_sort() -> Result<()> {
     let df_results = ctx
         .table("t1")
         .await?
-        .aggregate(vec![col("b")], vec![count(wildcard())])?
```

Comment:
Should we also deprecate `wildcard()` and Expr::Wildcard (in a follow on PR?)

https://docs.rs/datafusion/latest/datafusion/logical_expr/fn.wildcard.html
https://docs.rs/datafusion/latest/datafusion/prelude/enum.Expr.html#variant.Wildcard

🤔